x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -22,20 +29,23 @@ For {es}, adherence to FIPS 140-2 is ensured by
 === Upgrade considerations
 
 If you plan to upgrade your existing cluster to a version that can be run in
-a FIPS 140-2 enabled JVM, the suggested approach is to first perform a rolling
+a FIPS 140-2 configured JVM, the suggested approach is to first perform a rolling
```

Comment:
I think we can just go with `FIPS 140-2 JVM` from here out. Users should know what we're referencing.
```suggestion
a FIPS 140-2 JVM,  we recommend first perform a rolling
```

Reply:
I'll keep it for consistency and emphasis